frontend/src/components/common/GenericModal.tsx

```diff
@@ -163,23 +163,26 @@ export const GenericModal = (props: BsModalProps & ModalProps) => {
   );
 };
 
-const ModalContainer = (props: BsModalProps & ModalProps) =>
-  !props.asPopup ? (
+const ModalContainer = (props: BsModalProps & ModalProps) => {
+  const { modalSize, ...rest } = props;
+
+  return !props.asPopup ? (
```

Comment:
React was complaining about non-DOM property **modalSize** being set on a DOM element. This will fix that